src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
do not use static methods

Reply:
If I don't use the static User method, I'll get an error during compilation (java: non-static method findByEmail(java.lang.String) cannot be referenced from a static context)

Reply:
What approach do you need for invoke non-static method?

Reply:
don`t make this method static

Reply:
If you can`t resolve this problem, feel free ask in chat, mentors not answer in git